src/frontend/src/icons/googlemaps/index.tsx

```
@@ -0,0 +1,9 @@
+import React, { forwardRef } from "react";
+import GooglemapsIconSVG from "./googlemaps";
+
+export const GooglemapsIcon = forwardRef<
+  SVGSVGElement,
+  React.PropsWithChildren<{}>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)